src/docfx/config/Config.cs

```diff
@@ -31,7 +35,14 @@ internal sealed class Config
         /// <summary>
         /// Gets the file glob patterns excluded from this docset.
         /// </summary>
-        public readonly string[] Exclude = s_defaultExclude;
+        public readonly string[] Exclude = Array.Empty<string>();
+
+        /// <summary>
+        /// Gets the default file glob patterns excluded from this docset.
+        /// This typically contains system reserved exclude patterns,
+        /// users are recommended to use <see cref="Exclude"/> to exclude files.
+        /// </summary>
+        public readonly string[] DefaultExclude = s_defaultExclude;
```

Comment:
do you want to add a spec test for:
1. default exclude root folder/includes files event user specific the other `exclude`
2. user can overwrite the default exclude if they want to publish/build the root folder files

Reply:
`defaultExclude` has been limited to docfx reserved folders like `_site`, `localization`, which should not be overwrittern by user. It may confuse user whether to use `exclude` or `defaultExclude`, so we can only expose `exclude`

Reply:
No, `defaultExclude` should not contain `*`.